db/migrate/20160608162400_create_tags.rb

```diff
@@ -0,0 +1,7 @@
+class CreateTags < ActiveRecord::Migration
+  def change
+    create_table :tags do |t|
+      t.string :name, null: false
```

Comment:
I don't remember what you had on previous tables, but it's always good to have timestamps on your records. `t.timestamps` should go in pretty much every table.